packages/core/src/stores/user.ts

```diff
@@ -139,6 +139,27 @@ export const user = createStore((context) => {
           throw err
         }
       },
+      async copyRedefinePasswordLink(payload: { _id: string }) {
+        const store = useStore('user', context.manager)
+        const metaStore = meta(context)
+
+        const { error, result } = await (store.$functions.getRedefinePasswordLink({
```

Comment:
Couldn't this be rewritten as `store.$functions.getRedefinePasswordLink<typeof originalUser.functions.getRedefinePasswordLink>(..)`?